### ISSUE TITLE
implement bitrev

### ISSUE THREAD

**Issue by nplanel** [emoon/capstone#32]
```
-   30dc0:      00c1            bitrev %d1
+   30dc0:      00c1            .short 0x00c1
```

**Comment by emoon** [comment]
Never heard about this one. Is this some non-Motorola op?

**Comment by nplanel** [comment]
Yes I think : http://www.freescale.com/files/dsp/doc/ref_manual/CFPRM.pdf

bitrev : bit reverse
byterev : byte reverse
ff1 : first logical one

A quick look in the document, and they seems documented the ISA_A+, ISA_C somewhere
